apps/src/code-studio/components/progress/StudentTable.jsx

```diff
@@ -102,24 +106,25 @@ class StudentTable extends React.Component {
               onClick={() => onSelectUser(student.id)}
             >
               <td key={`td-${student.id}`} style={styles.td}>
-                <a href={this.getRowLink(student.id)} style={{width: '100%'}}>
-                  <div style={styles.studentTableRow}>
-                    {levels && (
-                      <TeacherPanelProgressBubble
-                        level={levels.find(
-                          level => student.id === level.user_id
-                        )}
-                      />
-                    )}
-                    <span
-                      style={
-                        levels ? styles.nameWithBubble : styles.nameInScript
-                      }
-                    >
-                      {student.name}
-                    </span>
-                  </div>
-                </a>
+                <div style={styles.studentTableRow}>
+                  {levels && (
+                    <TeacherPanelProgressBubble
+                      level={levels.find(level => student.id === level.user_id)}
+                    />
+                  )}
+                  <span
+                    style={levels ? styles.nameWithBubble : styles.nameInScript}
+                  >
+                    {student.name}
+                  </span>
+                  <a
```

Comment:
Note: I added this anchor tag + FontAwesome icon because we currently have two click handlers on a student's name in the student table: 

1. `onClick` handler on line 102
2. `<a>` tag on line 105

2 was introduced in #28859 because teachers wanted the ability to open a student's progress in a different tab. However, it was overriding 1 (the `onClick` handler that now allows us to reload progress asynchronously). My way of dealing with this was to add an "external link" icon next to each student's name in the table:

<img width="228" alt="Screen Shot 2019-07-24 at 6 43 10 PM" src="https://user-images.githubusercontent.com/9812299/61839902-31525800-ae44-11e9-92ba-e830933b25c4.png">

This allows teachers to still open a new tab if they want, but allows us to asynchronously reload as well.

Reply:
Clever solution!